### ISSUE TITLE
Can't read DLL

### ISSUE THREAD

**Issue by Maelep** [codemerx/CodemerxDecompile#68]
Hello,

I'm noob in development but I'm trying to find something similar to JustDecompile for MacOS so I tried your software because I think it's what I need but I don't manage to explore .DLL from uMod. Am I wrong about the purpose of this software?

All DLL can't be show because not a binary or unsupported text encoding.

**Comment by tsviatko** [comment]
This is exactly JustDecompile for MacOS. It doesn't get any better than that :)

Are you sure what you are trying to open is actually a .NET DLL? Cannot think of another reason it would not work.

**Comment by Maelep** [comment]
It's working with JustDecompile (on Windows) and it's also working with ILSpy on my MacOS.

I'm decompiling/exploring these files: https://umod.org/games/rust/download/develop

**Comment by atdimitrov** [comment]
Hey @Maelep. Which .dll file you experience problems with? Also, what is the exact nature of the problem, i.e. what is the behavior you are experiencing? Does the program crash? Or nothing happens when you select a file?

**Comment by Maelep** [comment]
Ok, I discovered the "error", I opened the folder that contains all the DLL's so it shows "All DLL can't be show because not a binary or unsupported text encoding.".

Opening all DLL's in the folder works.
Sorry for this useless issue.

**Comment by atdimitrov** [comment]
It's not useless, it's an actual issue. Thank you for letting us know :)

**Comment by Maelep** [comment]
Oh, ok :) I'm glad to help you to discover bugs and resolve them 😊

Does it should also opens DLLs in a folder that is in the source folder? I mean `mySourceFolder/anOtherFolder/someDLLs`

**Comment by atdimitrov** [comment]
Yes, it should traverse the whole directory tree and open all found files.
